local/src/test/scala/com/salesforce/op/local/OpWorkflowRunnerLocalTest.scala

```diff
@@ -117,4 +120,47 @@ class OpWorkflowRunnerLocalTest extends FlatSpec with PassengerSparkFixtureTest
     } score shouldBe expected
   }
 
+  it should "handle multi picklist features without throwing an exception" in {
```

Comment:
the tests seems an overkill for such a simple check ;) let's think if we can make is simpler and not so labor intensive.

Reply:
Let's simply test it in `FeatureTypeSparkConverterTest`